File: Baek_2693/src/baek2693.java
import java.util.Scanner;

public class baek2693 {
    public static void main(String args[]){

        Scanner sc = new Scanner(System.in);

        int T = sc.nextInt();


        for(int i=0; i<T; i++) {
            int max1 = 0;
            int max2 = 0;
            int max3 = 0;
            int[] A = new int[10];

            for(int j=0; j<10;j++) {
                A[j] = sc.nextInt();
            }
            for(int k=0; k<10; k++){
                if(A[k] > max1)
                    max1 = A[k];
            }

            for(int k=0; k<10; k++){
                if(A[k]>max2 && A[k]<max1)
                    max2 = A[k];
            }
            for(int k=0; k<10; k++){
                if(A[k]>max3 && A[k]<max2)
                    max3 = A[k];
            }

            System.out.println(max3);
        }





    }


}

File: Baek_2693/baek_2693.py
T = int(input())
A = []
max1 = 0
max2 = 0
max3 = 0

for i in range(T):
    for j in range(10):
        A.append(input())
    for k in range(10):
        if(A[k] > max1):
            max1 = A[k]
    for k in range(10):
        if(A[k]>max2 & A[k]<=max1):
            max2 = A[k]
    for k in range(10):
        if(A[k]>max3 & A[k]<=max2):
            max3 = A[k]

    print(max3)